specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/network/group.sasl/leader/client.rpt

```diff
@@ -0,0 +1,165 @@
+#
```

Comment:
Note: each of the other network script categories indicate the versions of the Kafka wire protocol api(s) in use, even if multiple - for example `unmerged.p3.f5.d0.m5`. 😄 

Suggest doing the same for `group` category, also for `group.sasl` category.